finders/bixi/parser.py

```diff
@@ -1,24 +1,40 @@
 from typing import List
 import json
 from finders.gps import GPSPosition
-from finders.stops import Stop, BixiStop
+from finders.stops import BixiStopInformation, BixiStopStatus
+import requests
```

Comment:
Add a requirements.txt file and a requests in it, you will need to adjust the travis yaml file too